docs/plugins/export.rst

```diff
@@ -63,3 +69,15 @@ The default options look like this::
                 indent: 4
                 separators: [',' , ': ']
                 sort_keys: true
+        csv:
+            formatting:
+                ensure_ascii: False
+                indent: 0
+                separators: [',']
+                sort_keys: true
+        xml:
+            formatting:
+                ensure_ascii: False
+                indent: 4
+                separators: ['>']
+                sort_keys: true
```

Comment:
I believe these examples are now out of sync with the implementation. It would also be good to describe the options in English above.